docs/cromwell_features/HogFactors.md

```diff
@@ -0,0 +1,116 @@
+# Hog Factors
+
+As of Cromwell 35, Cromwell introduces the concept hog factor (along with the related of hog groups and hog limits).
+
+## Concepts
+
+### Hog Group
+
+- Every workflow is assigned to a hog group when Cromwell receives it. Exactly how this happens is configurable.
+- Every sub-workflow and job triggered by that workflow is associated with the same hog group as its parent.
+
+The limits discussed below happen at the granularity of the hog group.
+
+### Hog Factor
+
+The hog factor is an integer greater than or equal to 1. It represents a trade-off between: 
```

Comment:
Styling note: add 1 more space after the colon so that the unordered list after displays correctly. Currently:
<img width="898" alt="screen shot 2018-09-21 at 1 25 40 pm" src="https://user-images.githubusercontent.com/2395211/45896350-df632080-bda1-11e8-8e31-6430488b95ce.png">
Should be: 
<img width="881" alt="screen shot 2018-09-21 at 1 26 15 pm" src="https://user-images.githubusercontent.com/2395211/45896377-f1dd5a00-bda1-11e8-88a6-b88fe58f8eac.png">